### ISSUE TITLE
Is this still in development?

### ISSUE THREAD

**Issue by ShalokShalom** [qmlweb/qmlweb#424]
Since there are 20 commits since one year: Can I consider this project as abadoned?

**Comment by stephenmdangelo** [comment]
I wouldn’t say it’s so much abandoned as it implements enough of a subset of QML for the current users’ purposes. At least that’s how I see it. 

Is there particular functionality you are looking for that is missing?

**Comment by ShalokShalom** [comment]
I once posted a random example into the online client and it reported half of the used libs as unsupported. 
PureQML is in active development..

**Comment by stephenmdangelo** [comment]
Are you able to provide the example you used that failed?

Re: PureQML, I have not used it, but my usecase was to reuse the same QML code for a native C++ application and a web application so qmlweb seemed like a better fit at the time.

**Comment by ShalokShalom** [comment]
No, I did that several years ago. It was from the Qt website.

**Comment by ChALkeR** [comment]
@ShalokShalom Yes, this is still in development. :wink: 
I was absent for a while due to personal burn-out (which could be observed on my GitHub activity, but I'm fine now.

I'm a bit disappointed that no one else from @qmlweb/collaborators maintained this in my absense, but that probably was partially my fault. @qmlweb/collaborators, I abolished the previous pr review process, so feel free to fix/commit/accept PRs on your own judgment. Waiting for reviews on complex and/or controversial things would still be good, but again — on your own judgment ;-). Given what I observed now, it would be easier for us to fix things later than to being idle and doing nothing here. I closed #46, btw.

@ShalokShalom Now, to PureQML — I tried that personally, but I failed to run even simple testcases from our [tests dir](https://github.com/qmlweb/qmlweb/tree/master/tests/Render/Simple). Did you succeed in running anything? Asking for personal reasons :wink:. Note that I also added other [related efforts](https://github.com/qmlweb/qmlweb/blob/master/README.md#related-efforts) to the Readme, perhaps some of those would be of more use to you right now.

tl;dr — no, QmlWeb is not abadoned, and I plan to continue working on it. Does that answer your question?

**Comment by ShalokShalom** [comment]
Wonderful. ^-^

I prefer your project since it is compatible with vanilla QML. 

One thing I like to mention is the interop with the Qt Quick Designer, since you can basically draw homepages with that. 

The fundamental issue with lots of shiny new and sane projects is very often the propaganda. 
Can you imagine to create a little showcase? 

I think about a small little video which starts from scratch, involves the Qt Quick Designer and deploys to your library, maybe including the installation process. 

I see here a chance to show how quick and elegant you can deploy.

I did not try PureQML yet, do you consider merging some parts of their code?
In order to achieve the same power as with qmlweb, would be a fork needed for the Qt Quick Designer and frontend code cant be reused from desktop and mobile applications. 

All that can be solved by a source compiler, while all this increases the work to be done.

**Comment by qt-br** [comment]
Hi! Is this project active?

**Comment by Plaristote** [comment]
Not really. I keep an eye on new issues, but we haven't made any changes in a while.
It's still functional, but while we've got the most essential features, we've only implemented a small subset of Qml.
